javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,117 @@
+let name = ""
+const todoList = []
+
+function getReply(command) {
+ if (typeof command !== "string" || command === "") {
+   return "Please enter a valid command"
+ }
+/* let num1 = x;
+let num2 = y;*/
+ switch(command){
+
+   case "Hello my name is Benjamin":
+     name = "Benjamin"
```

Comment:
The command should work for any name, not just `Benjamin` ie it should work with your name also 
`getReply("Hello my name is Shahnaz");`